src/main.ts

```diff
@@ -120,6 +120,8 @@ async function run(): Promise<void> {
     }
 
     if (trivyStatus == trivyHelper.TRIVY_EXIT_CODE) {
+        utils.createScanReport();
+        core.setOutput('scan-report', utils.getScanReportPath());
```

Comment:
You could take the report's path as an output from `utils.createScanReport()`

Reply:
Done